src/nunit.gui.core/Model/TestModel.cs

```diff
@@ -202,12 +198,12 @@ public void NewProject()
 
         public void NewProject(string filename)
         {
-            MessageBox.Show("It's not yet decided if we will implement saving of projects. The alternative is to require use of the project editor, which already supports this.", "Not Yet Implemented");
+            throw new NotImplementedException();
```

Comment:
Why this change? I understand the gist of it, but I don't see where we are catching this type of exception to show the user. So when this is called by the MainPresenter would this cause an unhandled exception to occur. Same for `SaveProject`

Reply:
Good catch. I intended to move the message display to the presenter. This gets System.Windows.Forms out if the model.